miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/StorageLocationRestController.java

```diff
@@ -84,7 +84,8 @@ private StorageLocationDto doSave(StorageLocationDto dto) {
   }
 
   @PostMapping(value = "/freezers/{id}/shelves")
-  public @ResponseBody StorageLocationDto addFreezerShelf(@PathVariable(name = "id", required = true) long id) {
+  public @ResponseBody StorageLocationDto addFreezerShelf(@PathVariable(name = "id", required = true) long id,
+      @RequestParam(name = "identificationBarcode", required = false) String barcode) {
     StorageLocation freezer = getFreezer(id);
     int lastShelf = freezer.getChildLocations().stream().filter(loc -> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove useless curly braces around statement and then remove useless return keyword [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1602)